src/app/profile/payment-methods/payment-method/forms/add-payment-method/modal.tpl.html

```diff
@@ -0,0 +1,87 @@
+<div class="modal-content give-modal-content">
```

Comment:
I believe model-content class is added by the UI-bootstrap JS.  Double check that there isn't already a div with this class as a parent when the modal is open.